src/content/docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring.mdx

```diff
@@ -118,11 +125,11 @@ With our infrastructure monitoring tools you can also:
 * [Create, view, or update alert settings](/docs/add-edit-or-view-host-alert-information) directly from the relevant infrastructure chart. For example, you can create a [host not reporting](/docs/infrastructure/new-relic-infrastructure/configuration/create-infrastructure-host-not-reporting-condition) alert condition.
 * [Forward logs to our logs monitoring](/docs/logs/enable-new-relic-logs/1-enable-logs/forward-your-logs-using-new-relic-infrastructure-agent) by adding as many configuration files as log sources you need to push to our platform.
 
-## Install and configure the infrastructure agent [#install]
+## Install the infrastructure agent [#install]
 
-You can send [metric data](/docs/infrastructure/manage-your-data/data-instrumentation/default-infrastructure-events) to our platform using our open source infrastructure agent, a [lightweight](/docs/infrastructure/new-relic-infrastructure/getting-started/agent-performance-overhead) executable file that works in the background to collect the information you need.
+Our open source infrastructure agent is a [lightweight](/docs/infrastructure/new-relic-infrastructure/getting-started/agent-performance-overhead) executable file that works in the background to collect data from your operating system. The infrastructure agent comes bundled with our on-host integrations, which report data from a wide variety of infrastructure services like MySQL, Cassandra, NGINX, and many more. 
```

Comment:
I'm not sure I'd use bundled here. You can install infra as a stand-alone, without any integration.